Documentation/packaging.md

```diff
@@ -81,6 +60,34 @@ Packages using the `--with-stage1-flavors=host` option must therefore add a run-
 Whenever systemd and bash are upgraded on the host, rkt will use the new version at run time.
 It becomes the packager's responsibility to test the rkt package whenever a new version of systemd is packaged.
 
-# Other code bundle
+### Other build considerations
+
+#### Godep
+rkt uses [Godep](https://github.com/tools/godep) to maintain [a copy of dependencies in its source repository](https://github.com/coreos/rkt/tree/master/Godeps).
+
+## Run-time dependencies
+
+A Linux system configured with [suitable options](hacking.md#run-time-requirements) is required.
+
+### Additional run-time dependencies for the host flavor
+
+- systemd >= v222 with systemctl, systemd-shutdown, systemd, systemd-journald.
+- bash
+
+### Ownership and permissions of rkt directories
+
+In general, subdirectories of `/var/lib/rkt` should be created with the same ownership and permissions as done by `rkt install`, see [directory list](https://github.com/coreos/rkt/blob/master/rkt/install.go#L44).
+
+The package should create a group `rkt` and `/var/lib/rkt` should belong to group `rkt` with the `setgid` bit set (`chmod g+s`)
+
+When the ownership and permissions of `/var/lib/rkt` are set up correctly, users member of `rkt` should be able to fetch ACIs without using sudo.
+However, `root` is still required to run pods.
+
+### systemd units
```

Comment:
Not sure why the "systemd units" section is part of the "Run-time dependencies" section.